config/application.yml.example

```diff
@@ -57,6 +59,7 @@ test:
   allow_third_party_auth: 'yes'
   domain_name: 'www.example.com'
   ducksboard_uid: '8VEQre2c8b62mQV0sa'
+  email_from: 'no-reply@login.gov'
```

Comment:
Whan values don't differ with the environment you can just put them at the top.  see `support_url` for example